cmd/ltctl/comparison.go

```diff
@@ -102,6 +103,10 @@ func writeReports(results []comparison.Result, outPath string) error {
 }
 
 func RunComparisonCmdF(cmd *cobra.Command, args []string) error {
+	if _, err := exec.LookPath("gnuplot"); err != nil {
+		return fmt.Errorf("gnuplot is not installed. The comparison command requires it to be installed: %w", err)
+	}
```

Comment:
The check looks good, but we should only do it if the value of `Output.GenerateGraphs` in the comparison config is true. So maybe it makes more sense to also move the check to `comparison.New`, after the default validation.